skytemple_files/_resources/ppmdu_config/pmd2data.xml

```diff
@@ -68,592 +68,9 @@
-        <Block name="ShopMainMenu1" beg="0x4740" end="0x4760"/>
-        <Block name="ShopMainMenu2" beg="0x4760" end="0x4780"/>
-        <Block name="ShopMainMenu3" beg="0x4780" end="0x47B0"/>
-      </Bin>
-      <Bin filepath="overlay/overlay_0023.bin" loadaddress="0x0238AC80">
-        <Block name="StorageMenuConfirm" beg="0x31BC" end="0x31D4"/>
-        <Block name="StorageMainMenu1" beg="0x31D4" end="0x31F4"/>
-        <Block name="StorageMainMenu2" beg="0x31F4" end="0x3214"/>
-        <Block name="StorageMainMenu3" beg="0x3214" end="0x3234"/>
-        <Block name="StorageMainMenu4" beg="0x3234" end="0x325C"/>
-      </Bin>
-      <Bin filepath="overlay/overlay_0024.bin" loadaddress="0x0238AC80">
-        <Block name="HatcherMenuConfirm" beg="0x23E0" end="0x23F8"/>
-        <Block name="HatcherMainMenu" beg="0x23F8" end="0x2418"/>
-      </Bin>
-      <Bin filepath="overlay/overlay_0025.bin" loadaddress="0x0238AC80">
-        <Block name="AppraiserMenuConfirm" beg="0x1374" end="0x138C"/>
-        <Block name="AppraiserMainMenu" beg="0x138C" end="0x13AC"/>
-        <Block name="AppraiserSubMenu" beg="0x13AC" end="0x13CC"/>
-      </Bin>
-      <Bin filepath="overlay/overlay_0026.bin" loadaddress="0x0238AC80"/>
-      <Bin filepath="overlay/overlay_0027.bin" loadaddress="0x0238AC80">
-        <Block name="DiscardItemsMenuConfirm" beg="0x281C" end="0x2834"/>
-        <Block name="DiscardItemsSubMenu1" beg="0x2834" end="0x2854"/>
-        <Block name="DiscardItemsSubMenu2" beg="0x2854" end="0x2874"/>
-        <Block name="DiscardItemsMainMenu" beg="0x2874" end="0x289C"/>
-      </Bin>
-      <Bin filepath="overlay/overlay_0028.bin" loadaddress="0x0238AC80"/>
-      <Bin filepath="overlay/overlay_0030.bin" loadaddress="0x02383420"/>
-      <Bin filepath="overlay/overlay_0031.bin" loadaddress="0x02383420">
-        <Block name="DungeonMainMenu" beg="0x75D8" end="0x7618"/>
-        <Block name="DungeonSubMenu1" beg="0x7674" end="0x7694"/>
-        <Block name="DungeonSubMenu2" beg="0x7694" end="0x76B4"/>
-        <Block name="DungeonSubMenu3" beg="0x76B4" end="0x76D4"/>
-        <Block name="DungeonSubMenu4" beg="0x76D4" end="0x76F4"/>
-        <Block name="DungeonSubMenu5" beg="0x7920" end="0x7938"/>
-        <Block name="DungeonSubMenu6" beg="0x79A4" end="0x79EC"/>
-      </Bin>
-      <Bin filepath="overlay/overlay_0032.bin" loadaddress="0x02383420"/>
-      <Bin filepath="overlay/overlay_0033.bin" loadaddress="0x02383420"/>
-      <Bin filepath="overlay/overlay_0034.bin" loadaddress="0x022DCB80">
-        <Block name="UnknownMenuConfirm" beg="0xD4C" end="0xD64"/>
-        <Block name="DungeonDebugMenu" beg="0xD74" end="0xD9C"/>
-      </Bin>
-      <Bin filepath="overlay/overlay_0035.bin" loadaddress="0x022BD3C0"/>
-      <!-- THIS IS END45'S 'EXTRA SPACE' OVERLAY. IT REQUIRES THE 'ExtraSpace' PATCH -->
-      <Bin filepath="overlay/overlay_0036.bin" loadaddress="0x023A7080"/>
-    </Game>
-
-    <!--Japan-->
-    <Game id="EoS_JP">
```

Comment:
I was lazy and haven't added the JP values in `pmdsky-debug`, but maybe we should before deleting this file lol

Reply:
I mean it's still always here in the history haha

Reply:
Decided to stop being lazy and added them :P